game/functions.js

```diff
@@ -1,4 +1,56 @@
-module.exports = { lastChar, testFisrtChar, formatString };
+require('dotenv').config();
+
+module.exports = { validateWord };
+
+async function validateWord(rawWord, messageAuthor, rawChars, lastAuthor, client) {
+	const word = await formatString(rawWord);
+
+	if (process.env.DEBUG === 'false' && messageAuthor === lastAuthor) {
+		return {
+			error: true,
+			message: 'It\'s not your turn!', 
+			chars: '',
+		};
+	}
+
+	if (await word.includes(' ')) {
+		return {
+			error: true,
+			message: 'Send only a single word please!',
+			chars: '',
+		};
+	}
+
+	if (rawChars !== '[]') {
+		if (!await testFisrtChar(word, await JSON.parse(rawChars))) {
+			return {
+				error: true,
+				message: 'Your word should start with the last letter of the previous word.\n**Note:** Someone might have been a prankster and deleted their word, run `/status` to check what letter\'s next.', 
```

Comment:
@kovagoadi Do you think this would give an idea to trolls?

Reply:
I think this is fine for now, we can always change it later.